pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1393,6 +1393,7 @@ func BenchmarkUpdate1kNodes30kPods(b *testing.B) {
 		cachedNodes := schedulernodeinfo.NewSnapshot()
 		cache.UpdateNodeInfoSnapshot(cachedNodes)
 	}
+	b.StopTimer() // without this line we're taking the overhead of defer() into account
```

Comment:
Hmm... Defer setting feature gate is not a time-costing operation.

To not confuse the readers, my humble option is we should use `b.StopTimer()` only when it's necessary - i.e. a heavy `defer()`.

Reply:
Emm... Thanks @Huang-Wei  
It is just a small improvement.
If this doesn't matter, let's do something else more important.
What important tasks are we doing about kube-scheduler? 
I'd like to help with anything I can.  ^_^